ustring.h

```diff
@@ -11,15 +36,33 @@
 #include <span>
 #include <locale>
 
+#if IS_STANDARDIZED
+
+#define STD std
```

Comment:
I've seen people do this before and I'm not sure I understand the benefit of it. This implementation doesn't need this kind of tight integration with the standard library. When things get adopted they often have to be rewritten to conform to the conventions and requirements of the library. It is more beneficial (IMO) to just use `stdx` as you have and not have ways to switch the `namespace` name with the pre-processor.